doc/changelog.rst

```diff
@@ -19,11 +36,33 @@ Backwards incompatible changes
 TODO: add examples of how to deal with these changes
 
 -  Allow contiguous representation of extracted regions (:pull:`2230`) by :user:`rebeccaherman1`
```

Comment:
```suggestion
-  Allow contiguous representation of extracted regions (:pull:`2230`) by :user:`rebeccaherman1`
   The preprocessor function :func:`esmvalcore.preprocessor.extract_region` no longer
   automatically maps the extracted :class:`iris.cube.Cube` to the 0-360 degrees 
   longitude domain. If you need this behaviour, use
   `cube.intersection(longitude=(0., 360.))` in your Python code after extracting the
   region. There is no possibility to restore the previous behaviour from a recipe.
```
Maybe add something along those lines?

Reply:
Thanks @bouweandela! 🎉 Done in 86950411c 😊